hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -57,11 +60,20 @@
 public class HoodieRealtimeTableCompactor implements HoodieCompactor {
 
   private static Logger log = LogManager.getLogger(HoodieRealtimeTableCompactor.class);
+  // Accumulator to keep track of total log files for a dataset
+  private AccumulatorV2<Long, Long> totalLogFiles;
```

Comment:
could we use Accumulators for IO handle metrics as well?

Reply:
We could use but that would require adding another first level parameter to WriteClient and then to upsert/insert table operation. Is that something that we want to introduce ? Reporting from executors might be overloading the metrics sink.

Reply:
any updates on this based on our discussion? I do find it weird to mix performance metrics with the same class holding file level metadata..